src/client/locales/en.json

```diff
@@ -298,5 +298,6 @@
   "login_modal_footer_text": "Don't have an account? Signup with {link}",
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
-  "login_with_steemconnect": "Login with SteemConnect"
-}
\ No newline at end of file
+  "login_with_steemconnect": "Login with SteemConnect",
+  "follows_you": "Follows you"
```

Comment:
What do we do with translations at the moment?
Should they be in en.json or default.json?

@bonustrack

Reply:
Translation should be in `default.json`

Reply:
Good catch, `en.json` should not be used anymore.

Reply:
should it be in both en.json and default.json? @bonustrack @Sekhmet

Reply:
For now yes @jm90m